docs/HyperIndex/Advanced/rpc-sync.md

```diff
@@ -29,13 +29,12 @@ Below is an example of how sync configuration per RPC endpoint can be defined:
 - id: 1
   rpc_config:
     url: https://eth.com # RPC URL that will be used to subscribe to blockchain data on this network
-    unstable__sync_config:
-      initial_block_interval: 10000 # Integer
-      backoff_multiplicative: 0.8 # Float
-      acceleration_additive: 2000 # Integer
-      interval_ceiling: 10000 # Integer
-      backoff_millis: 5000 # Integer
-      query_timeout_millis: 20000 # Integer
+    initial_block_interval: 10000 # Integer
+    backoff_multiplicative: 0.8 # Float
+    acceleration_additive: 2000 # Integer
+    interval_ceiling: 10000 # Integer
+    backoff_millis: 5000 # Integer
+    query_timeout_millis: 20000 # Integer
```

Comment:
It's just this I changed, the rest looks like formatting

Reply:
This could go in the migration guide possibly, but I think the migration guide is big enough and not many people were using this. So lets keep the migration guide as is.